theSrc/internal_www/data/displayr_regression/set9/colorscale_numeric_smallmultiples_state/after_drag.json

```diff
@@ -0,0 +1,58 @@
+{
+    "X": [
+      1,
+      0,
+      8,
+      0,
+      0,
+      2,
+      19,
+      0,
+      15,
+      1,
+      14,
+      7,
+      6
+    ],
+    "Y": [
+      1,
+      0,
+      4,
+      0,
+      0,
+      0,
+      5,
+      0,
+      8,
+      1,
+      5,
+      4,
+      3
+    ],
+    "label": [
+      "63",
+      "126",
+      "189",
+      "1008",
+      "1071",
+      "1134",
+      "1197",
+      "2016",
+      "2079",
+      "2142",
+      "3024",
+      "3087",
+      "3150"
+    ],
+    "hiddenlabel.pts": [],
+    "plotlyAnnotations": [
+      {
+        "index": 0,
+        "text": "63",
+        "visible": true,
```

Comment:
Is it necessary to store `text` and `visible`?

Reply:
Its necessary to store `visible` because the label can be toggled off. Its not strictly necessary to store `text` but it makes it easier to debug because otherwise it hard to tell which label goes with what.